css/gallery.css

```diff
@@ -1,107 +1,130 @@
 /*class for the section of gallery*/
+.gallery-item::before {
+  content: " ";
+  display: block;
+  padding-top: 100%;
+}
 .gallery {
-  width: 635px;
-  height: 88px;
   font-family: Roboto;
```

Comment:
This line is not necesary, font is set in general in styles.css, i know you did not add it here but some cleanup comes well